engine/src/main/scala/cromwell/engine/workflow/FileSystemWorkflowStoreDatabase.scala

```diff
@@ -0,0 +1,95 @@
+package cromwell.engine.workflow
+
+import java.io.File
+import java.nio.file.{Files, Path, Paths}
+import java.util.UUID
+
+import cromwell.core.{WorkflowId, WorkflowSourceFiles}
+import cromwell.engine.workflow.WorkflowStoreDatabase.{Restartable, StartableState, Submitted, WorkflowToStart}
+import org.apache.commons.io.FileUtils
+
+import scala.concurrent.{ExecutionContext, Future}
+import scala.language.postfixOps
+import scalaz.NonEmptyList
+
+case object FileSystemWorkflowStoreDatabase extends WorkflowStoreDatabase {
+
+  val separator = File.separator
+  val fsroot= Paths.get("WorkflowStore").toAbsolutePath
+  val submittedPath = Paths.get(s"$fsroot${separator}submitted").toAbsolutePath
+  val restartablePath = Paths.get(s"$fsroot${separator}restartable").toAbsolutePath
+  val runningPath = Paths.get(s"$fsroot${separator}running").toAbsolutePath
+
+  def workflowPath(base: Path, id: WorkflowId): Path = Paths.get(s"$base$separator$id")
+
+  def wdlFilePath(base: Path) = Paths.get(s"${base.toAbsolutePath}${separator}workflow.wdl")
+  def inputsFilePath(base: Path) = Paths.get(s"${base.toAbsolutePath}${separator}inputs.json")
+  def optionsFilePath(base: Path) = Paths.get(s"${base.toAbsolutePath}${separator}options.json")
```

Comment:
Those are needed, otherwise it would be `$separatoroptions.json`. IntelliJ is cool about it, I believe